src/services/study.ts

```diff
@@ -8,25 +8,18 @@
 import { UUID } from 'crypto';
 import { backendFetchJson } from './utils';
 
-const PREFIX_STUDY_QUERIES = import.meta.env.VITE_API_GATEWAY + '/study';
+const PREFIX_STUDY_QUERIES = `${import.meta.env.VITE_API_GATEWAY}/study`;
 
-export function exportFilter(
-    studyUuid: UUID,
-    filterUuid?: UUID,
-    token?: string
-) {
+function exportFilter(studyUuid: UUID, filterUuid?: UUID, token?: string) {
     console.info('get filter export on study root node');
     return backendFetchJson(
-        PREFIX_STUDY_QUERIES +
-            '/v1/studies/' +
-            studyUuid +
-            '/filters/' +
-            filterUuid +
-            '/elements',
+        `${PREFIX_STUDY_QUERIES}/v1/studies/${studyUuid}/filters/${filterUuid}/elements`,
         {
             method: 'get',
             headers: { 'Content-Type': 'application/json' },
         },
         token
     );
 }
+
+export default exportFilter;
```

Comment:
Given the structure, this one is a bit unexpected, maybe you can disable the rule for this one.